test/image/mocks/zindex-scatter-image.json

```
@@ -0,0 +1,23 @@
+{
+"data": [
+    {
+        "type": "scatter",
+        "y": ["E", "F", "G", "H"],
+        "x": ["A", "B", "C", "D"]
+        }, {
+        "type": "image",
+        "zmax": [1, 1, 1],
+        "x0": "B",
+        "y0": "F",
+        "z": [
+        [[1, 0, 0], [0, 1, 0], [0, 0, 1], [0, 0, 0]],
+        [[0, 1, 0], [0, null, 1], [1, 0, 0], [0, 0, 0]],
+        [[1, 0, 0], [1, 0, 0], [0, 0, 1], [0, 0, 0]]
+        ],
+        "zindex":10
+    }
+],
+    "layout": {
+        "width": 400, "height": 600
```

Let's reduce the size here for faster image comparison, etc.
```suggestion 
        "width": 400, "height": 400
```